utils/docker/publish_docker.sh

```diff
@@ -28,36 +23,20 @@ fi
 
 # Ubuntu 20.04
 FOCAL_TAGS=(
-  "next"
-  "next-focal"
   "v${PW_VERSION}-focal"
 )
 
-if [[ "$RELEASE_CHANNEL" == "stable" ]]; then
-  FOCAL_TAGS+=("focal")
-fi
-
 # Ubuntu 22.04
 JAMMY_TAGS=(
-  "next-jammy"
   "v${PW_VERSION}-jammy"
   "v${PW_VERSION}"
```

Comment:
So, our default tag is still jammy? Why recommend noble?

Reply:
Done.